### PR TITLE
Oss 125

### PR DESCRIPTION
**Pull request by adam-kicsak**
New custom service properties

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -81,25 +81,40 @@
         <extensions>true</extensions>
         <configuration>
           <instructions>
-            <Import-Package>org.everit.osgi.ecm.component;version="[1.0,2)",*</Import-Package>
-            <Export-Package>${project.artifactId}</Export-Package>
+            <Import-Package>
+              org.everit.osgi.ecm.component;version="[1.1,2)",
```
Comment (balazs-zsoldos) [inline core/pom.xml]: As ECM Component implements ECM API, it should import only range [1.1,1.2).

File: core/pom.xml [hunk 2]
```diff
@@ -81,25 +81,40 @@
...
+              org.everit.osgi.ecm.component.resource;version="[1.0,2)",
```
Comment (balazs-zsoldos) [inline core/pom.xml]: As ECM Component implements ECM API, it should import only range [1.0,1.1)

File: core/pom.xml [hunk 3]
```diff
@@ -81,25 +81,40 @@
...
+              *
+              </Import-Package>
+            <Export-Package>
+              org.everit.osgi.ecm.component.ri;version="1.2.3"
```
Comment (balazs-zsoldos) [inline core/pom.xml]: You sure it is 1.2.3?

File: core/pom.xml [hunk 4]
```diff
@@ -81,25 +81,40 @@
...
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
 
   <dependencies>
     <dependency>
       <groupId>org.everit.osgi</groupId>
       <artifactId>org.everit.osgi.ecm.component.api</artifactId>
-      <version>1.0.0</version>
+      <version>1.1.0-SNAPSHOT</version>
```
Comment (balazs-zsoldos) [inline core/pom.xml]: ecm 1.1.0 api has just been released so your pull request does not have to fail on Travis ;).

File: core/pom.xml (hunk 3 above)
Comment (adam-kicsak) [inline core/pom.xml]: Of course it is incorrect, this shuold be 1.3.0. I have been updated the versions based on your comments.

**Pull request merged**